### ISSUE TITLE
to hide status bar

### ISSUE THREAD

**Issue by caoyang1024** [Orderella/PopupDialog#126]
# Report

## Environment
Please provide information on your development environment, so we can build with the same scenario.

- Mac OS version (e.g. 10.12): 10.12.3
- Xcode version (e.g. 8.0): 8.2.1
- PopupDialog version (e.g. 0.5.0): 0.5.0
- Minimum deployment target (e.g. 9.0): 9.0
- Language (Objective-C / Swift): swift
- In case of Swift - Version (e.g. 3.0): 3.0

## Dependency management
If you are not using any dependency managers, you can remove this section.
> Please note: If you are using CocoaPods with Xcode 8, CocoaPods 1.1.0 is required.

- Dependency manager (e.g. CocoaPods): CocoaPods
- Version (e.g. 1.1.0): 1.1

## What did you do?

try to hide the status bar

## What did you expect to happen?

// to hide status bar
```
override var prefersStatusBarHidden: Bool {
        return true
}
```

## What happened instead?

status bar will show now

## Project that demonstrates the issue

n.a

**Comment by mwfire** [comment]
Good point. I will consider this for the next release, thanks!

**Comment by caoyang1024** [comment]
That's great!

**Comment by mwfire** [comment]
Released with V0.6.1
